downloads/tests/settings/19.py

```diff
@@ -0,0 +1,97 @@
+import os
+from os.path import expanduser
+
+SECRET_KEY = "PLACE_SECRET_KEY_HERE"
+
+USE_TZ = True
+
+TIME_ZONE = "Africa/Johannesburg"
+
+DATABASES = {
```

Comment:
Use postgres please. We're moving all test to it because it actually enforces referential integrity.